Cookbook/Technical-Documents/PullAssigners.md

```diff
@@ -0,0 +1,104 @@
+
+ - Require pull request reviews before merging
+   - Required approving reviews: 2
+
+## Additional possibilities
+
+### If you want to always require at least one review from specific list of owners
+
+If you want to enforce your PRs to always be approved someone from a smaller, fixed list of people (like the "main owners of the repo"), in addition to the people Pull Assigner will assign randomly to all new PRs, you could also add a team of those people as owners of your codebase, and use "Require review from Code Owners" on your protected branch.
+
+That's what we use for our bot repos `Stevenson` and `Wall-E`, as we want the always have at least one person from the Developer Experience squad –who is responsible for maintaining the bots– to approve PRs in those bot repositories, in addition to Pull Assigner randomly assigning other people from the rest of the team to distrubute review work.
+
+To achieve a similar setup for your repository:
+
+1. Create a GitHub team containing the owners of your repository's codebase (i.e. add to that team the members from which you want at least one approval on all the PRs).  
+   _For example, for our bot repositories `Stevenson` and `Wall-E`, we created a team called `ios-bot-owners` containing the people from the Developer Experience Squad, as we want each PR to be approved by at least one person from this team/squad before it can be merged, in addition to at least another reviewer from the ones picked by Pull Assigner_
+
+2. Again, make sure that you add your repository to the ones that new team has access to (e.g. via [this page for `ios-bot-owners`](https://github.com/orgs/Babylonpartners/teams/ios-bot-owners/repositories))
+3. Add that new team to your `.github/CODEOWNERS` file, alongside the proxy team you already added:  
+
+    ```
+    # Global rule for the whole codebase.
+    #  - Every PR needs to be reviewed by at least one member of the `ios-bot-owners` team.
+    #  - The empty `iOS-PullAssigner` team will act as a proxy for the PullAssigner bot.
+    #    When a new PR arrives, PullAssigner will be triggered, which will then pick members from the iOS-Admin team
+    
+    * @Babylonpartners/iOS-PullAssigner @Babylonpartners/ios-bot-owners
+    ```
+
+4. Go back to your repository's settings in GitHub, under the protected branches (see previous section), and check the checkbox for "Require review from Code Owners"
+
+This way, no PR could be merged unless at least one person from the teams listed in `CODEOWNERS` approves it, and in this case since the `iOS-PullAssigner` team is empty, that means at least one person from `ios-bot-owner` has to approve the PR – even if 2 other people affected to the PR by Pull Assigners already approved it
```

Comment:
```suggestion
This way, no PR could be merged unless at least one person from the teams listed in `CODEOWNERS` approves it, and in this case since the `iOS-PullAssigner` team is empty, that means at least one person from `ios-bot-owner` has to approve the PR – even if 2 other people assigned to review by Pull Assigners already approved it
```